src/components/pages/CategoriesTable.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Table } from 'antd';
+
+function CategoriesTable() {
+  // const [data, setData] = useState([]);
```

Comment:
Just from following what Ryan said last week, we probably don't need the commented out code. But we will need this line (or something similar) once we're getting actual data so it doesn't really bother me too much!